packages/cursorless-engine/src/actions/ShowParseTree.ts

```diff
@@ -22,7 +22,7 @@ export default class ShowParseTree {
       results.push(parseTree(editor.document, tree, contentRange));
     }
 
-    ide().openUntitledTextDocument({
+    void ide().openUntitledTextDocument({
```

Comment:
should we not await this?

Reply:
huh interesting. I wonder when that promise actually resolves. maybe let's at least leave a FIXME

Reply:
Agreed.